packages/api/src/keymanager/routes.ts

```diff
@@ -166,16 +175,40 @@ export type Api = {
   ): Promise<{
     data: ResponseStatus<DeleteRemoteKeyStatus>[];
   }>;
+
+  listFeeRecipient(
+    pubkey: string
+  ): Promise<{
+    data: FeeRecipientData;
+  }>;
+  setFeeRecipient(pubkey: string, ethaddress: string): Promise<void>;
+  deleteFeeRecipient(pubkey: string): Promise<void>;
+
+  getGasLimit(
+    pubkey: string
+  ): Promise<{
+    data: GasLimitData;
+  }>;
+  setGasLimit(pubkey: string, gasLimit: string): Promise<void>;
+  deleteGasLimit(pubkey: string): Promise<void>;
 };
 
 export const routesData: RoutesData<Api> = {
   listKeys: {url: "/eth/v1/keystores", method: "GET"},
   importKeystores: {url: "/eth/v1/keystores", method: "POST"},
-  deleteKeystores: {url: "/eth/v1/keystores", method: "DELETE"},
+  deleteKeys: {url: "/eth/v1/keystores", method: "DELETE"},
 
   listRemoteKeys: {url: "/eth/v1/remotekeys", method: "GET"},
   importRemoteKeys: {url: "/eth/v1/remotekeys", method: "POST"},
   deleteRemoteKeys: {url: "/eth/v1/remotekeys", method: "DELETE"},
+
+  listFeeRecipient: {url: "/eth/v1/validator/{pubkey}/feerecipient", method: "GET"},
+  setFeeRecipient: {url: "/eth/v1/validator/{pubkey}/feerecipient", method: "POST"},
+  deleteFeeRecipient: {url: "/eth/v1/validator/{pubkey}/feerecipient", method: "DELETE"},
+
+  getGasLimit: {url: "/eth/v1/validator/{pubkey}/gas_limit", method: "GET"},
```

Comment:
should this be called `listGasLimit` to be consistent with the naming of other routes?

Reply:
this is how the spec does it, and with the spec matcher there isn't much we can do :shrug: https://github.com/ethereum/keymanager-APIs/blob/master/apis/gas_limit.yaml#L2 as we have been trying to directly use operationId as our route name.